SampleApps/CarthageTest/CarthageTest/ViewController.swift

```diff
@@ -31,7 +31,7 @@ class ViewController: UIViewController {
             apiClient: apiClient,
             universalLink: URL(string: "https://mobile-sdk-demo-site-838cead5d3ab.herokuapp.com/braintree-payments")!
         )
-        let sepaDirectDebitClient = BTSEPADirectDebitClient(apiClient: apiClient)
+        let sepaDirectDebitClient = BTSEPADirectDebitClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
nit: can we use the `authorization` constant here instead of the actual string value